### PR TITLE
Improve code flow & performance

### PR DESCRIPTION
**Pull request by pgilad**
Also change to using included files by npm instead of ignoring

### CODE REVIEW

**Comment by ben-eb** [comment]
Should we introduce new dependencies? lodash's find is slightly faster than native methods, sure, but a for loop is faster still; http://jsperf.com/native-filter-vs-lodash-find/7

Other modules that need the `uniq` method use https://github.com/fgnass/uniqs - so would prefer this as it doesn't add more weight to browser builds.

Perhaps for cssnano 4 we should assess usage of dependencies in more detail? For now I'd like to be consistent with what we already use. :smile:

The rest looks good to me. :+1:

**Comment by TrySound** [comment]
I wouldn't like to use extra dep. Performance issue between lodash and [].filter is not very sharp. So better use native method.

File: package.json [hunk 1]
```diff
@@ -7,6 +7,12 @@
     "lint": "jshint index.js lib/*.js --reporter node_modules/jshint-stylish/stylish.js",
     "test": "tape test.js | tap-spec"
   },
+  "files": [
+      "CHANGELOG.md",
```
Comment (TrySound) [inline package.json]: Usually we do not include changelog.

**Comment by pgilad** [comment]
Filter isn't the correct function to use, what you need is `forEach` with breaking when you find the correct item.
Regarding `uniqs` - it's not robust enough to pass a predicate function so you would need to use a flatten array. I don't mind converting the usage here to flat arrays, but why bother ? I trust lodash perfs over any other library because I know they are 100% tested and battle-tested.

Regarding `changelog` - will remove it

**Comment by TrySound** [comment]
@pgilad Partial methods from lodash contains swarm of deps. Usually in my node_modules both and bundle and these partials. It's sad and big.

**Comment by pgilad** [comment]
I removed `lodash.find` and kept `lodash.uniq`. I don't really feel like re-writing unique, and this isn't browser code where every byte counts..

**Comment by ben-eb** [comment]
Looks good to me; although using this setup we don't actually need to use a predicate function. We can instead convert a flat array of numbers into `from` & `to` objects once `optimizeValues` is called. Simpler, I think, and doesn't add more dependencies than what we already are using. I think proper benchmarking and an assessment of dependencies is needed for v4.

Anyway, thanks a lot for your contribution, much appreciated! :+1:

**Pull request merged**